### DISCUSSION TITLE
Suggest a new feature/idea

### DISCUSSION THREAD

**Discussion by UltimaHoarder** [UltimaHoarder/UltimaScraper#583] [Ideas]
You can suggest ideas here.

If I decide to work on it, I'll pin it here 👍 
-----------------------------
Just make a new thread instead.

**Comment by joujoucoons** [comment]
download multiple accounts at same time.
i mean when it asks for you to input a number for account you could input more than one "3, 4, 5" etc

**Reply by James-MF-Thomas** [reply]
sounds like a whitelist... 🤔

**Reply by joujoucoons** [reply]
where can i find it?

**Reply by UltimaHoarder** [reply]
@joujoucoons I implemented this with the latest commit

**Comment by avekifes** [comment]
Is it possible to rip comments that users make on posts?

**Reply by DonaldTPP** [reply]
Why the hell would you want to waste time doing that?

**Reply by UltimaHoarder** [reply]
@avekifes I will add it as an option to do so

@ProjektPremium There are many machine learning use cases for this

**Comment by vctess** [comment]
I am using docker, but I am stuck after running the images. 
Please add a tutorial on youtube using this tool on docker. I think this will help many people using this tool if there is a tutorial using docker.
Thank you.

**Reply by UltimaHoarder** [reply]
Sorry, I don't know how to use Docker. The docker features are only there because someone contributed to the readme.
